index.html

```diff
@@ -1321,6 +1321,14 @@ <h2 id="BaseAudioContext">
             </p>
           </dd>
           <dt>
+            ConstantSourceNode createConstantSource()
```

Comment:
Yeah, I did that for consistency since we have, of course, factory methods for all other nodes. I'm ok with leaving this out.